src/Symfony/Component/Yaml/Tests/DumperTest.php

```diff
@@ -507,6 +512,9 @@ public function testDumpingNotInlinedNullTaggedValue()
 YAML;
 
         $this->assertSame($expected, $this->dumper->dump($data, 2));
+        $this->assertEquals(
+            ['foo' => new TaggedValue('bar', 'null')],
+            $this->parser->parse($expected, Yaml::PARSE_CUSTOM_TAGS | Yaml::PARSE_CONSTANT));
```

Comment:
Why adding tests for the parsing in the DumperTest ? That's quite confusing to me.

Reply:
I want to verify that our expected yaml output is correct, and the best way to do this seems to be a round trip.
On the one hand this is not really the responsibility of this test, on the other hand it is more useful than it hurts - imo.